include/rocksdb/statistics.h

```diff
@@ -304,6 +304,17 @@ enum Tickers : uint32_t {
   // # of bytes in the blob files evicted because of BlobDB is full.
   BLOB_DB_FIFO_BYTES_EVICTED,
 
+  // These coutners indicate a performance issue in WritePrepared transactions.
+  // We should not seem them ticking them much.
+  // # of times prepare_mutex_ is acquired in the fast path.
+  TXN_PREPARE_MUTEX_OVERHEAD,
```

Comment:
by "overhead" I thought it mean time spent, but they are just counts. Should we call them "xxx_count" instead?

Reply:
But we do not count it all the time; only when they are acquired on the fast path.